guides/reactjs-relay.md

```diff
@@ -82,16 +82,18 @@ function App() {
 # Connect to Other Peers
 
 The Waku instance needs to connect to other peers to communicate with the network.
-First, create `bootstrapWaku` to connect to the Status fleet:
+First, create `bootstrapWaku` to connect to Waku bootstrap nodes (hosted by Status):
```

Comment:
I'm not sure we need to mention hosted by Status, in the future with DNS discovery we'll run these ourselves and then it is just arbitrary nodes.

In general, any mention of Status at this layer is a code/doc smell

Reply:
At the moment, we do not bootstrap to a decentralized set of nodes run by a various parties, the bootstrapping grabs nodes addresses from fleet.status.im and connect to them.

For the sake of transparency, I believe it is important to state it.

I agree that with DNS discovery, this should go away and dependence on Status infrastructure will reduce.

Reply:
@oskarth do you want to react to that? I am keen to have your input re transparency.

Reply:
Personally I'd do it as I suggested in OP, and I think having a comment above those nodes re cluster would be enough for transparency at this stage. If you feel strongly that this parenthesis should be there that's fine with me. The important thing is that we don't enshrine anything Status specific, and also that we remove it once we get more nodes (which we should all actively work on doing).